controlled_navigation/public/html/controlled_navigation.html

```diff
@@ -13,3 +13,44 @@
         {% endif %}
     </div>
 </div>
+
+<!--
+This is a simplified version of this script:
+https://github.com/openedx/edx-platform/blob/master/lms/templates/courseware/courseware-chromeless.html#L119-L217
+
+The script is responsible for resizing the iframe containing the controlled navigation block.
+By default, in edx-platform, the script is loaded in the courseware-chromeless.html template,
+however, in chromium based browsers, the script is not executed when navigating between child
+components. So, we need to include the script manually.
+-->
+<script type="text/javascript">
+    (function () {
+        if (window !== window.parent) {
+            document.body.className += " view-in-mfe";
+            var contentElement = document.getElementById("content");
+
+            function dispatchResizeMessage(event) {
+                var newHeight = contentElement.offsetHeight;
+                var newWidth = contentElement.offsetWidth;
+
+                window.parent.postMessage(
+                    {
+                        type: "plugin.resize",
+                        payload: {
+                            width: newWidth,
+                            height: newHeight,
+                        },
+                    },
+                    "*"
```

Comment:
can you include in the comment that we're modifying this?

Reply:
Done: https://github.com/eduNEXT/xblock-controlled-navigation/commit/0adcf6732803c72aeaaaa23661d77f386bb11520